react-web/src/pages/auditing/reportUpload/reportUpload.schema.tsx

```diff
@@ -0,0 +1,97 @@
+import * as yup from "yup";
+
+export const FILE_SIZE = 5;
+export const SUPPORTED_FORMATS = [".pdf"];
+export const allowedTypes = ["application/pdf"];
+
+export const reportUploadSchema = yup.object().shape({
+  certificationLevel: yup.string().required("This field is required"),
+  summary: yup
+    .string()
+    .required("This field is required")
+    .max(200, "Please enter less than 200 characters"),
+  disclaimer: yup
+    .string()
+    .required("This field is required")
+    .max(200, "Please enter less than 200 characters"),
```

Comment:
```suggestion
```

Reply:
There is no maximum required in the CIP-0096 spec, I just checked. The requirement comes from the DApp store which we're going to push on them to do something about that.